tools/bazel.rc

```diff
@@ -2,5 +2,8 @@
 build --verbose_failures
 test --test_output=errors
 
+# Release Build
+build:release -c opt
```

Comment:
yes, envoy is special. let's first check if there's any difference with rules_go. if not, then we can't even use bazel to accomplish optimized build.